developers/node-tutorial.md

```diff
@@ -237,9 +237,13 @@ commands:
 3. Build the `celestia` binary:
 
    ```bash
-   make build
+   make build-jemalloc
    ```
 
+   ::: tip NOTE
+   This build option enables CGO. [Learn more about the build command](https://github.com/celestiaorg/celestia-node/releases/tag/v0.12.1#:~:text=%F0%9F%8F%97%EF%B8%8F-,New%20build%20option,-%3A%20Makefile%20now%20has).
```

Comment:
Another thing I just realized is that this build downloads and installs jemalloc. It doesn't hurt to mention that.

Reply:
got it! following up in PR